atomisticparsers/gromacs/parser.py

```diff
@@ -559,12 +617,28 @@ def __init__(self):
         self.log_parser = GromacsLogParser()
         self.traj_parser = GromacsMDAnalysisParser()
         self.energy_parser = GromacsEDRParser()
-        self._metainfo_mapping = {
-            'LJ (SR)': 'Leonard-Jones', 'Coulomb (SR)': 'coulomb',
-            'Potential': 'potential', 'Kinetic En.': 'kinetic'}
         self._frame_rate = None
         # max cumulative number of atoms for all parsed trajectories to calculate sampling rate
         self._cum_max_atoms = 2500000
+        self._gro_energy_units = ureg.kilojoule * MOL
+        self._thermo_maps = {}
+        self._thermo_maps['ignore_list'] = ['Time', 'Box-X', 'Box-Y', 'Box-Z']
+        self._thermo_maps['dir_to_ind_map'] = {
+            'XX': np.array([0, 0]), 'XY': np.array([0, 1]), 'XZ': np.array([0, 2]),
+            'YX': np.array([1, 0]), 'YY': np.array([1, 1]), 'YZ': np.array([1, 2]),
+            'ZX': np.array([2, 0]), 'ZY': np.array([2, 1]), 'ZZ': np.array([2, 2])}
+        self._thermo_maps['base_calc_map'] = {
```

Comment:
i do not quite like the idea of putting all the maps in thermo_maps because they have different vallues. Also for the base_calc_map, since this is a fixed key and unit, I suggest using tuple insted of dict.